kerbauth_test.go

```diff
@@ -0,0 +1,301 @@
+package mssql
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+
+	"gopkg.in/jcmturner/gokrb5.v7/client"
+	"gopkg.in/jcmturner/gokrb5.v7/config"
+	"gopkg.in/jcmturner/gokrb5.v7/credentials"
+	"gopkg.in/jcmturner/gokrb5.v7/keytab"
+	"gopkg.in/jcmturner/gokrb5.v7/messages"
+	"gopkg.in/jcmturner/gokrb5.v7/spnego"
+	"gopkg.in/jcmturner/gokrb5.v7/types"
+)
+
+func createKrbFile(filename string) string {
+	ans := []byte{84, 104, 105, 115, 32, 105, 115, 32, 97, 32, 116, 101, 115, 116, 32, 102, 105, 108, 101, 46}
```

Comment:
I generally don't like unit tests, but if they need to, pass "t *testing.T" as first parameter and fail the test on error, but don't just print something.

Reply:
There should be some explanation of what this byte string is.

Reply:
I just realized this isn't a binary file, but a text file that contains "This is a test file.". Do do this, write the following code:

ans := []byte("This is a test file.")